examples/linearAerodynamics/linearAerodynamics.m

```diff
@@ -149,15 +152,15 @@
 %md
 %md For such propose the angle of incidence and the wind properties are computed as:
 % then characteristic dimension is extracted executing: 
-dimCaracteristic = norm(elements(2).elemTypeAero (1:3) ) ;
+dimCaracteristic = norm(elements(2).chordVector ) ;
 % the angle of attack is: 
-betaRel = acos(dot(elements(2).elemTypeAero( 1:3 ) , [0 0 1] ));
+betaRel = acos(dot(elements(2).chordVector , [0 0 1] ));
 % the wind velocity is:
 windVel = feval(analysisSettings.fluidProps{3,:}, betaRel, analysisSettings.finalTime) ;
 % Extract the aerodynamic coefficients 
-userDragCoef   = elements(2).aeroCoefs{1,:} ; 
-userLiftCoef   = elements(2).aeroCoefs{2,:} ; 
-userMomentCoef = elements(2).aeroCoefs{3,:} ; 
+userDragCoef   = "dragCoefFunctionLA"; 
```

Comment:
I suggest always using ' instead of "  for the sake of compatibility between octave and matlab ...  https://www.mathworks.com/matlabcentral/answers/485385-difference-between-single-quote-vs-double-quote

Reply:
Perfect, done